enhancements/installer/component-selection.md

```diff
@@ -393,7 +393,7 @@ Would expect this to go directly to GA once a design is agreed upon/approved.
     Review presubmit results for your annotation pull request to ensure that:
     1. The capability is being completely removed, without leaving dangling resources that you forgot to annotate.
     2. CI suites which you expected to pass continue to pass.
-        You may find that some test-cases assume or require your capability's presence, and they may need to grow logic to skip or alter the test conditions when your capability is not installed (like [this][bump-origin]).
+        You may find that some test-cases assume or require your capability's presence, and they may need to grow logic to skip or alter the test conditions when your capability is not installed (like [this][bump-origin] or by adding a suitable regex to the `[Skipped:NoOptionalCapabilities]` [annotation rule][annotation-rules].
```

Comment:
nit: you've dropped the trailing `)`, leaving parens unbalanced.

Reply:
thanks, fixed